sdk/textanalytics/ai-text-analytics/test/public/textAnalyticsClient.spec.ts

```diff
@@ -764,16 +764,16 @@ matrix([["AAD", "APIKey"]] as const, async (authMethod: AuthMethod) => {
             );
           });
 
-          // it("family emoji wit skin tone modifier", async function(this: Context) {
-          //   await checkOffsetAndLength(
-          //     client,
-          //     "👩🏻‍👩🏽‍👧🏾‍👦🏿 SSN: 859-98-0987",
-          //     "Utf16CodeUnit",
-          //     25,
-          //     11,
-          //     checkEntityTextOffset
-          //   );
-          // });
+          it("family emoji wit skin tone modifier", async function(this: Context) {
+            await checkOffsetAndLength(
+              client,
+              "👩🏻‍👩🏽‍👧🏾‍👦🏿 SSN: 859-98-0987",
```

Comment:
What's been changed?
How are the recordings working just by uncommenting the tests?
Can you elaborate more on the changes?

Reply:
The fix is in the recording file itself, see @witemple-msft commit.

Reply:
I see. Looks fine.
Can we do better? Seems predictable, we can automate to modify the request bodies I guess.

Reply:
Yes, we already have a discussion about patching the recorder in the SIG chat.